pegasus/sites.v3/code.org/public/js/admins_email_signup_form.js

```diff
@@ -0,0 +1,17 @@
+// Code provided from Google reCAPTCHA
+function timestamp() { var response = document.getElementById("g-recaptcha-response"); if (response == null || response.value.trim() == "") {var elems = JSON.parse(document.getElementsByName("captcha_settings")[0].value);elems["ts"] = JSON.stringify(new Date().getTime());document.getElementsByName("captcha_settings")[0].value = JSON.stringify(elems); } } setInterval(timestamp, 500);
+
+setInterval(timestamp, 500);
+
+// Set a disabled attribute on button until
+// the reCAPTCHA checkbox is clicked
+function recaptchaCallback() {
+  const btnSubmit = document.querySelector("button");
+
+  if (btnSubmit.hasAttribute("disabled")) {
+    btnSubmit.removeAttribute("disabled");
+    btnSubmit.setAttribute("disabled");
+  }
+}
+
+recaptchaCallback();
```

Comment:
Does this need to be called here, or is referencing it in the HTML enough?

Reply:
I was getting a linting error when I tried committing w/o it.

Reply:
My hunch is that we don't need to run the function here –– it should get executed at the right time via the HTML `data-callback`

Reply:
Removed!

Reply:
Ah, sorry, I missed your comment before I wrote mine above! Sounds great –– thank you!